test/remote/db_tests.js

```diff
@@ -644,24 +644,6 @@ describe('remote db', function() {
     }
   )
 
-  it(
-    'account deletion',
-    () => {
-      return db.emailRecord(ACCOUNT.email)
-        .then(function(emailRecord) {
-          assert.deepEqual(emailRecord.uid, ACCOUNT.uid, 'retrieving uid should be the same')
-          return db.deleteAccount(emailRecord)
-        })
-        .then(function() {
-          // account should no longer exist for this email address
-          return db.accountExists(ACCOUNT.email)
-        })
-        .then(function(exists) {
-          assert.equal(exists, false, 'account should no longer exist')
-        })
-    }
-  )
-
```

Comment:
This is just moved down so that the signinCodes tests happen before we delete the account.